build.gradle

```diff
@@ -40,8 +40,8 @@ configure(subprojects.findAll { it.name != 'util' }) {
     apply plugin: 'optional-base'
 
     group = 'org.mongodb.morphia'
-    sourceCompatibility = JavaVersion.VERSION_1_6
-    targetCompatibility = JavaVersion.VERSION_1_6
+    sourceCompatibility = JavaVersion.VERSION_1_8
```

Comment:
1.6 is the maximum JVM version we're requiring at this point.  Many of our users are still on older versions and bumping this would break them.